source/sass/components/_ui-elements.scss

```diff
@@ -0,0 +1,18 @@
+
+.fram-welcome {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  .fram-welcome__logo {
```

Comment:
using sass to make sorter way to write fram-welcome__logo

Reply:
Using sass nesting selector, but consider refer to this article: https://www.sitepoint.com/beware-selector-nesting-sass/